api/envoy/config/route/v3/route_components.proto

```diff
@@ -1573,7 +1573,7 @@ message DirectResponseAction {
       "envoy.api.v2.route.DirectResponseAction";
 
   // Specifies the HTTP response status to be returned.
-  uint32 status = 1 [(validate.rules).uint32 = {lt: 600 gte: 100}];
+  uint32 status = 1 [(validate.rules).uint32 = {lt: 600 gte: 200}];
```

Comment:
I think it's valid in this case to make the breaking change with a stricter PGV validation, since there's no reasonable way it could have been returning 1xx before (it was broken in Envoy and arguably any other proxy). Pinging @envoyproxy/api-shepherds in case there is any other thinking here.